config/gulp/gulp.browser-sync.js

```diff
@@ -0,0 +1,31 @@
+var gulp = require('gulp');
+var webpack = require('webpack');
+var webpackDevMiddleware = require('webpack-dev-middleware');
+var webpackHotMiddleware = require('webpack-hot-middleware');
+var browserSync = require('browser-sync');
+var webpackDevConfig = require('../webpack/webpack.dev.conf');
+var bundler = webpack(webpackDevConfig);
+
+gulp.task('browser-sync', function() {
+  browserSync.init({
+    port: 8004,
+    ui: {
+      port: 8006
```

Comment:
Why not 8005?

Reply:
http://www.networksorcery.com/enp/protocol/ip/ports08000.htm

it's used for some little thing.. thought about port 8006 because hes totally free

Reply:
No idea what this is, but okay